### PR TITLE
DOC-6451: Updated the External Secrets Service section of the Corda 5.3 deployment instructions

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Preview: https://anna.preview.docs.r3.com/en/platform/corda/5.3/deploying-operating/deployment/deploying.html#using-hashicorp-vault-in-the-openshift-environment

### CODE REVIEW

File: content/en/platform/corda/5.3/deploying-operating/deployment/deploying/_index.md [hunk 1]
```diff
@@ -1051,6 +1051,19 @@ For example, when running with Red Hat OpenShift Container Platform, you must us
         name: "corda-privileged"
    ```
 
+4. Override client image for bootstrap vault populating job with specific values recommended by Hashicorp. You must provide this configuration in your `values.yaml` file. Here is an example:
```
Comment (anton-subbotin) [inline content/en/platform/corda/5.3/deploying-operating/deployment/deploying/_index.md:1054]: This isn't related to the Service Accounts; I suggest that this is moved to under the "External Secrets Service" section, possibly under a dedicated sub-header, describing that this is only related to the the Openshift environment

**Review by anton-subbotin — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.3/deploying-operating/deployment/deploying/_index.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.3/deploying-operating/deployment/deploying/_index.md]: Updated

**Review by anton-subbotin — DISMISSED** [review]
LGTM

File: content/en/platform/corda/5.3/deploying-operating/deployment/deploying/_index.md [hunk 2]
```diff
@@ -892,6 +892,21 @@ These keys are not tied to the schema names. If the schema names change, the key
 {{< /note >}}
 Additionally, a passphrase and salt for the Corda [wrapping keys]({{< relref "../../../key-concepts/cluster-admin/_index.md#key-management" >}}) must be added to the Vault `cryptosecrets` path under the keys `passphrase` and `salt` respectively.
 
+##### Using HashiCorp Vault in the OpenShift Environment
+
+When deploying Corda Enterprise on Red Hat OpenShift using HashiCorp Vault as an external secret service, you must override client image for bootstrap vault populating job with specific values recommended by HashiCorp. You must provide this configuration in your `values.yaml` file. Here is an example:
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.3/deploying-operating/deployment/deploying/_index.md:897]:
```suggestion
When deploying Corda Enterprise on Red Hat OpenShift using HashiCorp Vault as an external secret service, you must the override client image for the bootstrap vault populating job with specific values recommended by HashiCorp. You must provide this configuration in your `values.yaml` file. Here is an example:
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

**Review by paulmoloneyr3 — APPROVED** [review]

File: content/en/platform/corda/5.3/deploying-operating/deployment/deploying/_index.md [hunk 3]
```diff
@@ -892,6 +892,21 @@ These keys are not tied to the schema names. If the schema names change, the key
 {{< /note >}}
 Additionally, a passphrase and salt for the Corda [wrapping keys]({{< relref "../../../key-concepts/cluster-admin/_index.md#key-management" >}}) must be added to the Vault `cryptosecrets` path under the keys `passphrase` and `salt` respectively.
 
+##### Using HashiCorp Vault in the OpenShift Environment
+
+When deploying Corda Enterprise on Red Hat OpenShift using HashiCorp Vault as an external secret service, you must the override client image for the bootstrap vault populating job with specific values recommended by HashiCorp. You must provide this configuration in your `values.yaml` file. Here is an example:
```
Comment (anton-subbotin) [inline content/en/platform/corda/5.3/deploying-operating/deployment/deploying/_index.md:897]: I'm a bit late, but `you must the override client image` doesn't seem right..

**Pull request merged**